docs/source/contributing/development.rst

```diff
@@ -235,7 +235,7 @@ Further useful guidelines
    frequently.
 
 #. When opening a new issue, if there are old issues that are related, link
-   them in your new issue (even if the old ones are closed).
+   them to your new issue (even if the old ones are closed).
```

Comment:
When opening a new issue, if there are old issues that are related, add a link to them in your new issue (even if the old ones are closed).